src/Compilers/VisualBasic/Portable/Symbols/SignatureOnlyMethodSymbol.vb

```diff
@@ -162,7 +162,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
 
         Public Overrides ReadOnly Property TypeArguments() As ImmutableArray(Of TypeSymbol)
             Get
-                Throw ExceptionUtilities.Unreachable
+                Return ImmutableArray(Of TypeSymbol).Empty
```

Comment:
>Return ImmutableArray(Of TypeSymbol).Empty [](start = 16, length = 42)

Why is this code reachable now? In any case this property should return _typeParameters.